### ISSUE TITLE
Admin\Operator - Internal User can view "Operator details"

### ISSUE THREAD

**Issue by zoeyli-46** [bcgov/cas-registration#1699]
#### Description:
As an internal user, I can see the details for an operator

#### Acceptance Criteria:

Given I am on the [operator grid](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=7861-1060&t=EdbpdHgRnvGxWEpV-4)
When I click on view details for an operator
Then I am brought the operator details page for that operator
[Figma](https://www.figma.com/design/LsDmLDCdnJqI8UahyqvOD8/BCIERS-Designs?node-id=14652-3185&t=b1Z2o0PV01yDoPhP-4)

Given I am on the operator details page,
Then the title says "[Operator Name] Operator Details
So that I know which operator I am viewing details for

#### Development Checklist:

- [x] create new dashboard for [Operator Name] Operator Details
- [x] add tiles for each of Information, Operations, Contacts, Users (Events can be ignored for now)
- [x] add routing protections so that only approved internal users are able to access this dashboard
- [x] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies
#1697 
- Blocked by
- Blocking

**Comment by andrea-williams** [comment]
for clarity:
at the moment, when an internal user clicks on the "View Details" link next to an operator name in the operators grid, they only see this:

![Image](https://github.com/user-attachments/assets/ca616f70-1a46-4c33-9ef6-bb145e7bb26e)

instead of the Operator Details dashboard as shown in the linked wireframe
